docs/release-notes/2.2.0.md

```diff
@@ -3,3 +3,4 @@
 - Introduce PodDisruptionBudget for Ory Oathkeeper. For more information, see the [PR](https://github.com/kyma-project/api-gateway/pull/853).
 - Extend APIRule access strategies with `no_auth` to restrict accessible methods, thereby minimizing the exposure of the API.
 - Introduce consistent labeling for the Kyma modules. For more information, see the [documentation](https://kyma-project.io/#/api-gateway/user/00-10-overview-api-gateway-controller?id=labeling-resources).
+- Use `Warning` state instead of `Error` when there is more than one APIGateway CR on the cluster. For more information, see the [PR](https://github.com/kyma-project/api-gateway/pull/882).
```

Comment:
```suggestion
- Use the `Warning` state instead of `Error` when there is more than one APIGateway CR on the cluster. For more information, see the [PR](https://github.com/kyma-project/api-gateway/pull/882).
```